api.js

```diff
@@ -22,3 +22,13 @@ export const getArticleComments = ({ article_id }) => {
     return response;
   });
 };
+
+export const patchVotes = ({ article_id }) => {
+  return instance
+    .patch(`/articles/${article_id}`, {
+      inc_votes: 1,
```

Comment:
This is always incrementing the vote by 1, what about when you need to decrease the votes because of a downvote?